gridded_etl_tools/utils/convenience.py

```diff
@@ -632,4 +635,41 @@ def span_to_timedelta(cls) -> np.timedelta64:
             cls.SPAN_MONTHLY: np.timedelta64(1, "M"),
             cls.SPAN_YEARLY: np.timedelta64(1, "Y"),
         }
-        return span_to_td[cls.temporal_resolution()]
```

Comment:
Was this meant to be changed

Reply:
yes because we realized basing this on the temporal resolution didn’t always make sense.

In reality we are no longer using this class method in live code but it seems useful enough to warrant preservation for the time being.

Reply:
Yep will do